trunk/source/UltimateRegressionTest/src/de/uni_freiburg/informatik/ultimate/regressiontest/AbstractRegressionTestSuite.java

```diff
@@ -110,17 +117,82 @@ public Collection<UltimateTestCase> createTestCases() {
 		final Predicate<File> filesRegexFilter = getFilesRegexFilter();
 		for (final Config runConfiguration : runConfigurations) {
 			final Collection<File> inputFiles = getInputFiles(filesRegexFilter, runConfiguration);
-
+			final NestedMap3<String, String, String, String> skippedTests = getSkippedTests(runConfiguration);
 			for (final File inputFile : inputFiles) {
 				final UltimateRunDefinition urd =
 						new UltimateRunDefinition(inputFile, runConfiguration.getSettingsFile(),
 								runConfiguration.getToolchainFile(), getTimeout(runConfiguration, inputFile));
-				rtr.add(buildTestCase(urd, getTestResultDecider(urd)));
+				final String overridenVerdict = skippedTests.get(inputFile.getName(),
+						runConfiguration.getSettingsFile().getName(), runConfiguration.getToolchainFile().getName());
+				rtr.add(buildTestCase(urd, getTestResultDecider(urd, overridenVerdict)));
 			}
 		}
 		return rtr;
 	}
 
+	private static NestedMap3<String, String, String, String> getSkippedTests(final Config runConfiguration) {

```

Comment:
Maybe it would be good to document in a comment how the skip files work. It's not entirely clear to me.
- From this code, it seems like I can have `skip.yml` in the directory of the settings file, the directory of the toolchain file, or both.
- If I have both, they are processed in different orders depending on which path is longer -- does that mean one can in some sense overwrite the other?
- I cannot have `skip.yml` in any other directory, e.g. a subfolder that contains the particular program?
- If the program (resp. the toolchain resp. the settings file) is not in the same folder as the `skip.yml`, do I have to use relative paths in `skip.yml` ?

Reply:
I was planning to allow multiple skip-files and resolve them hierarchically (i.e., if a task occurs in different files, use the one that is closest to the task itself). But maybe this is simpler, if require the skip-files to occur only next to the file itself.